src/applications/appeals/995/utils/submit.js

```diff
@@ -132,14 +132,36 @@ export const getPhone = formData => {
     : {};
 };
 
+export const TEMP_DATE = '2006-06-06';
```

Comment:
How did we pick this as the TEMP_DATE?

Reply:
Randomly chosen. I originally used `2000-01-01` but then I already had that used in a bunch of unit tests and needed something different.